doc/api/n-api.md

```diff
@@ -420,7 +420,8 @@ typedef void (*napi_threadsafe_function_call_js)(napi_env env,
 - `[in] env`: The environment to use for API calls, or `NULL` if the thread-safe
 function is being torn down and `data` may need to be freed.
 - `[in] js_callback`: The JavaScript function to call, or `NULL` if the
-thread-safe function is being torn down and `data` may need to be freed.
+thread-safe function is being torn down and `data` may need to be freed,
```

Comment:
Nit: Maybe change this to "... freed. It may also be `NULL` if ..."

Reply:
Sounds better to me. Updated :)